<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutBorneStationnement</class>
 <widget class="QDialog" name="AjoutBorneStationnement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutBorneStationnement</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant  de la borne :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdVoiePublique">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Latitude">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IdBorne">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant Voie Publique : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom Topographique : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NomTopo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Longitude">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de la borne Stationnement : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumStatio">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cote Rue:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CoteRue">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>530</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Créer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Ok</sender>
   <signal>clicked()</signal>
   <receiver>AjoutBorneStationnement</receiver>
   <slot>valider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valider()</slot>
 </slots>
</ui>
